deployments/warp_routes/TRUMP/solanamainnet-trumpchain-config.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0xD84981Ecd4c411A86E1Ccda77F944c8f3D9737ab"
+    chainName: trumpchain
+    coinGeckoId: official-trump
```

Comment:
nit: supplying this for a token will mean that we consider it as "value at risk" and also have our dashboards consider it TVL, do you think it makes sense? for e.g. Lumia and inEVM we didn't specify it on the side that is pre-funded, so I think to be consistent we should do the same here